Makefile

```diff
@@ -21,14 +11,20 @@ ifneq ($(.SHELLSTATUS),0)
 	BRANCH := $(BRANCH)-dirty
 endif
 
-ifeq ($(BRANCH),develop)
-  DOCKER_IMAGE_REPO := go-spacemesh
-else
-  DOCKER_IMAGE_REPO := go-spacemesh-dev
-endif
+DOCKER_HUB ?= spacemeshos
+DOCKER_IMAGE_REPO ?= go-spacemesh-dev
```

Comment:
for prod we should use go-spacemesh, and keep `dev` only for dev builds.

Reply:
That's why I changed it 😉

Reply:
With this change every build on every branch is pushed to `go-spacemesh-dev` and tagged with the commit-sha. Releases are pushed to `go-spacemesh` and tagged with the git tag.

Before every commit on `develop` was pushed to `go-spacemesh` with the sha as tag and everything else (including releases) to `go-spacemesh-dev` also with the commit sha as image tag